datamodules/cifar100.py

```diff
@@ -28,27 +31,15 @@ def setup(self, stage: Optional[str] = None):
             os.getcwd(),
             train=True,
             download=True,
-            transform=T.Compose(
-                [
-                    T.Resize(size=(224, 224)),
-                    T.ToTensor(),
-                    T.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
-                ]
-            ),
+            transform=self.train_transforms_conf,
         )
         test = CIFAR100(
             os.getcwd(),
             train=False,
             download=True,
-            transform=T.Compose(
-                [
-                    T.Resize(size=(224, 224)),
-                    T.ToTensor(),
-                    T.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
-                ]
-            ),
+            transform=self.test_transforms_conf,
         )
-        train, val = random_split(train, [40000, 10000])
+        train, val = random_split(train, [45000, 5000])
```

Comment:
It would better if we should give configuration for this like instead of letting exact number 
`length = [int(len(dataset) * train_size), int(len(dataset) * val_size)]`
`train, val = random_split(dataset, length)`

Reply:
Alright